### PR TITLE
Rename scripts directory to bin

### PR DESCRIPTION
**Pull request by MauricioFauth**
- Related to #18512

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
Can we have bin/internal/ for scripts that are only internal and not needed in the final vendor bundles ?

**Review by williamdes — APPROVED** [review]
Can we have bin/internal/ for scripts that are only internal and not needed in the final vendor bundles ?

**Comment by MauricioFauth** [comment]
> Can we have bin/internal/ for scripts that are only internal and not needed in the final vendor bundles ?

Aren't all scripts internal? We remove this directory when creating a release.

**Comment by williamdes** [comment]
> > Can we have bin/internal/ for scripts that are only internal and not needed in the final vendor bundles ?
> 
> Aren't all scripts internal? We remove this directory when creating a release.

No they are not
And we should not remove the directory on some type of bundles, the source one I think
That said such bundles do not need internal scripts like check release excludes or create release

File: bin/lang-cleanup.sh
Comment (williamdes) [inline bin/lang-cleanup.sh:1]: Internal

File: bin/locales-contributors
Comment (williamdes) [inline bin/locales-contributors:1]: internal

File: bin/set-version.sh
Comment (williamdes) [inline bin/set-version.sh:1]: internal

File: bin/upload-release
Comment (williamdes) [inline bin/upload-release:1]: very internal

File: bin/transformations_generator_plugin.sh
Comment (williamdes) [inline bin/transformations_generator_plugin.sh:1]: internal

File: bin/check-release-excludes.sh
Comment (williamdes) [inline bin/check-release-excludes.sh:1]: very internal

**Comment by MauricioFauth** [comment]
> No they are not
> And we should not remove the directory on some type of bundles, the source one I think
> That said such bundles do not need internal scripts like check release excludes or create release

Besides the transformations scripts (which should be removed in my opinion, and replaced by proper interfaces), all scripts are internal. That's why I don't see the point of having the internal directory.

I changed the create release script to not remove the bin directory for the source bundle. Should I also do that for 5.2?

**Comment by williamdes** [comment]
> > No they are not
> > And we should not remove the directory on some type of bundles, the source one I think
> > That said such bundles do not need internal scripts like check release excludes or create release
> 
> Besides the transformations scripts (which should be removed in my opinion, and replaced by proper interfaces), all scripts are internal. That's why I don't see the point of having the internal directory.

Okay. For example in https://salsa.debian.org/phpmyadmin-team/phpmyadmin if I had the generate mo script it could re build the mo files from source too

> I changed the create release script to not remove the bin directory for the source bundle. Should I also do that for 5.2?

no, that's okay

**Pull request merged**